test/presubmit-test-kfp-kubernetes-library.sh

```diff
@@ -15,11 +15,20 @@
 
 source_root=$(pwd)
 
+if [ $(id -u) = 0 ]; then
+  apt-get update && apt-get install -y protobuf-compiler
+  # rust needed for transitive deps in dev extras on Python:3.12
+  apt-get install rustc -y
+else
+  sudo apt-get update && sudo apt-get install -y protobuf-compiler
+  # rust needed for transitive deps in dev extras on Python:3.12
+  sudo apt-get install rustc -y
+fi
```

Comment:
We don't want to keep compatibility with Prow since this is a migration. The Prow and GHA tests won't co-exist. That said, we can remove this if/else.

Reply:
Done